include/error.h

```diff
@@ -57,8 +57,10 @@
  * Last error from errno.h is __ELASTERROR 2000 . After it, can be declared
  * user errors
  */
+#define __ELASTERROR 2000
 #define EOVERRUN	(__ELASTERROR + 1) /* Circular buffer overrun */
```

Comment:
I would still go with my suggestion. 
Or use:
#ifndef __ELASTERROR 
#define __ELASTERROR 2000
#endif

Otherwise in some platforms a warning with __ELASTERROR already defined will show